packages/lib/src/components/Dropin/components/DropinComponent.tsx

```diff
@@ -130,7 +131,7 @@ export class DropinComponent extends Component<DropinComponentProps, DropinCompo
                     <div className={`adyen-checkout__dropin adyen-checkout__dropin--${status.type}`}>
                         {isRedirecting && status.props.component && status.props.component.render()}
                         {isLoading && status.props && status.props.component && status.props.component.render()}
-                        {elements && !!elements.length && (
```

Comment:
Sure will add.

Reply:
Added tests. Still need to mock something because of this line https://github.com/Adyen/adyen-web/blob/ffd80012b8d6495f82aadf8e22d646552ea99b50/packages/lib/src/components/Dropin/components/DropinComponent.tsx#L30
It's waiting for promise to resolve, otherwise the `instantPaymentElements` array stays empty.